Documentation/community/roadmap.rst

```diff
@@ -0,0 +1,218 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+Roadmap
+=======
+
+This page describes a high-level view of the main priorities for the project,
+agreed as a result of collaboration between Cilium's committers_ and the
+broader
+community. You'll also find here some pointers on how you can :ref:`influence
+the roadmap<rm-influence>`. 
+
+Major Feature Status
+--------------------
+
+
++-----------------------------+------------------------------------------------------------+
+| eBPF Networking             | Stable (:ref:`Roadmap Details<rm-advanced-networking>`)    |
+++----------------------------+------------------------------------------------------------+
+|| Kubernetes CNI             | Stable                                                     |
+++----------------------------+------------------------------------------------------------+
+|| Load Balancing             | Stable                                                     |
+++----------------------------+------------------------------------------------------------+
+|| Network Policy             | Stable                                                     |
+++----------------------------+------------------------------------------------------------+
+|| Kube-proxy Replacement     | Stable                                                     |
+++----------------------------+------------------------------------------------------------+
+| Multi-Cluster (ClusterMesh) | Stable (:ref:`Roadmap Details<rm-clustermesh>`)            |
++-----------------------------+------------------------------------------------------------+
+| Hubble Observability        | Stable (:ref:`Roadmap Details<rm-hubble-observability>`)   |
++-----------------------------+------------------------------------------------------------+
+| Service Mesh                | Stable (:ref:`Roadmap Details<rm-cilium-service-mesh>`)    |
```

Comment:
It will be great to have something like Stable from version 1.x.x, something like upstream k8s docs has.

Also, for service mesh, is the plan to mark it as stable from 1.12?

Reply:
I like the idea, but let's add this as an improvement on a future update so that we don't hold this up